docs/src/main/paradox/.htaccess

```diff
@@ -2,19 +2,8 @@ RedirectMatch 301 ^(.*)/scala/(.*) $1/$2?language=scala
 RedirectMatch 301 ^(.*)/java/(.*) $1/$2?language=java
 
 RedirectMatch 301 ^(.*)/stream/stages-overview\.html.* $1/stream/operators/index.html
-RedirectMatch 301 ^(.*)/project/migration-guide-1\.3\.x-2\.0\.x\.html.* $1/project/migration-guide-old.html
```

Comment:
these redirects no longer seemed relevant, as I also decided to remove the migration guides for specific Akka versions